saos-import/src/main/java/pl/edu/icm/saos/importer/commoncourt/judgment/remove/SourceCcRemovedJudgmentsFinder.java

```diff
@@ -0,0 +1,83 @@
+package pl.edu.icm.saos.importer.commoncourt.judgment.remove;
+
+import java.util.List;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import pl.edu.icm.saos.importer.commoncourt.judgment.download.SourceCcjExternalRepository;
+import pl.edu.icm.saos.persistence.model.SourceCode;
+import pl.edu.icm.saos.persistence.repository.JudgmentRepository;
+
+/**
+ * Finder of common court judgments that was removed
+ * from source external repository but still are present in the
+ * saos repository.
+ * 
+ * @author madryk
+ */
+@Service
+public class SourceCcRemovedJudgmentsFinder {
+
+    private final static Logger log = LoggerFactory.getLogger(SourceCcRemovedJudgmentsFinder.class);
+
+
+    private SourceCcjExternalRepository sourceCcjExternalRepository;
+    
+    private JudgmentRepository judgmentRepository;
+    
+    
+    private int pageSize = 1000;
+    
+    
+    //------------------------ LOGIC --------------------------
+    
+    /**
+     * Returns list of source judgment ids from saos repository
+     * that was removed from source external repository 
+     */
```

Comment:
Returns a list of common court judgment source ids that have been removed in the external source repository.